web/feedback/feedback-en.html

```diff
@@ -0,0 +1,123 @@
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta content="7 days" name="revisit-after">
+    <meta content="global" name="distribution">
+    <meta content="14 years" name="rating">
+    <meta content="de" name="content-language">
+    <meta content="Recycling" name="page-topic">
+    <meta content="all" name="robots">
+    <meta content="JundM100 Development" name="author">
+    <meta content="JundM100 Development" name="copyright">
+
+    <!-- file local changes -->
+    <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+          name="viewport"/>
+    <link href="favicon.png" rel="icon" type="image/png"/>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab&amp;family=Source+Sans+Pro&amp;display=swap"
+          rel="stylesheet">
+    <title>Recycling Feedback Form | JundM100 Development</title>
+    <script src="../script/index.js"></script>
+
+    <script type="text/javascript"
+            src="https://cdn.jsdelivr.net/npm/@emailjs/browser@3/dist/email.min.js">
+    </script>
+    <script type="text/javascript">
+   (function(){
+      emailjs.init("rI5Rup9M40uuxFGCc");
+   })();
+</script>
+    <style>
+
+/* Style the submit button with a specific background color etc */
+.button {
+  background-color: #48b47c;;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  width: 180px;
+  margin-left: 20px;
+}
+
+input {
+    width: 100%;
+    margin: 8px 0;
+    box-sizing: border-box;
+}
+
+
+textarea {
+    width: 100%;
+    height: 150px;
+    padding: 12px 20px;
+    box-sizing: border-box;
+    resize: vertical;
+}
+
+body {
+    font-family: "Source Sans Pro", sans-serif;
+}
+
+.container {
+    border-radius: 5px;
+    background-color: #f2f2f2;
+    padding: 20px;
+}
+    </style>
```

Comment:
Move style to a stylesheet (css) file and link it instead of embedding it in html code